### ISSUE TITLE
localkube should not use /usr/lib/systemd/system for service unit files

### ISSUE THREAD

**Issue by easeway** [kubernetes/minikube#2098]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): BUG REPORT

Please provide the following details:

**Environment**:

**Minikube version** : v0.22.3 (commit id: 33584026c1bbe92310c66926abfe3afe39381dd7)
- **OS** : Ubuntu 17.04
- **VM Driver** : none
- **ISO version** :
- **Install tools**:
- **Others**:

**What happened**:
E1020 10:45:30.305707   13466 start.go:150] Error starting host: Error starting stopped host: mkdir /usr/lib/systemd/system/: permission denied.

**What you expected to happen**:
On ubuntu, service unit files are under /lib/systemd/system.
The right place to create systemd unit files should be /etc/systems/system which is unified across all systemd based distributions.

**How to reproduce it** (as minimally and precisely as possible):
minikube start --vm-driver=none

**Output of `minikube logs` (if applicable)**:

**Anything else do we need to know**:

```
./pkg/minikube/constants/constants.go:140:	LocalkubeServicePath = "/usr/lib/systemd/system/localkube.service"
./pkg/drivers/none/none.go:153:	if err := os.MkdirAll("/usr/lib/systemd/system/", os.FileMode(0755)); err != nil {
```

The constant `LocalkubeServicePath` should be `/etc/systemd/system` which is consistent across different distributions.
In `none.go`, the constants should be used instead of a string literal. Actually, `mkdir` is not necessary here.

**Comment by dlorenc** [comment]
Fixed by #2099

**Comment by garykrige** [comment]
Is there a temp fix aside from building from source?
